ui/app/styles/core/element-styling.scss

```diff
@@ -195,8 +195,3 @@ label {
     cursor: not-allowed;
   }
 }
-
-// TODO remove when HDS has released fix
-.hds-form-masked-input .hds-form-masked-input__toggle-button {
```

Comment:
🤣 this was the whole reason for this upgrade....there was some scope creep a bit. Although these deprecations I resolved weren't going to be a problem until 5.0 I thought I'd get ahead of them now!